modules/default.nix

```
@@ -3,6 +3,7 @@
     ./build-tarball.nix
     ./docker-desktop.nix
     ./interop.nix
+    ./nvidia-container-toolkit.nix
```

```suggestion
    ./nvidia.nix
```
lets name this just nvidia, incase we collect more nvidia things